src/de.rs

```diff
@@ -0,0 +1,166 @@
+use std::collections::hash_map::Iter;
+
+use hrana_client_proto::Value;
+use serde::{
+    de::{value::SeqDeserializer, IntoDeserializer, MapAccess, Visitor},
+    Deserialize, Deserializer,
+};
+
+use crate::Row;
+
+/// Deserialize from a [`Row`] into any type `T` that implements `Deserialize`.
```

Comment:
I worry about the discoverability of this. Should we add examples of using this to the `Row` rustdoc? I'd say yes!

Reply:
I haven't done anything ux side I just wanted to get a POC working.

Reply:
I've added this to the example and added a doc example.